### DISCUSSION TITLE
Virus Detection by Google Chrome/Windows defender

### DISCUSSION THREAD

**Discussion by Nostalgia-3** [bridge-core/bridge.#212] [Q&A] [answered]
**Summary**
I cannot get the bridge@1.6.0.exe file via https://github.com/bridge-core/bridge./releases/tag/v1.6.0 and don't know what to do.

**To Reproduce**
1. Go to https://github.com/bridge-core/bridge./releases/tag/v1.6.0
2. Click on bridge@1.6.0.exe

**Observed behavior**
The file does not get downloaded, it instead makes windows defender say that they found a virus, and my browser (google chrome) had also detected a virus
**Expected behavior**
To install and run the program

**Screenshots / File Attachments**
![Image1](https://user-images.githubusercontent.com/51009128/83962226-888ccb80-a850-11ea-9ae5-dd8c1561e090.png)
![Image2](https://user-images.githubusercontent.com/51009128/83962229-89bdf880-a850-11ea-9f00-7cd1b059cb97.png)

**Platform (please complete the following information):**
 - OS: Windows 10
 - App Version: 1.6.0

**Additional context**
N/A

**Comment by ENDERZOMBI102** [comment, marked as answer]
You can permit the download in the actions dropdown, and, in case that doesn’t work, disable defender before downloading the file and then re-enable it
